cluster/addons/fluentd-gcp/fluentd-gcp-configmap.yaml

```diff
@@ -366,6 +366,11 @@ data:
       </record>
     </filter>
 
+    # Do not collect fluentd's own logs to avoid infinite loops.
+    <match fluent.**>
```

Comment:
Does that mean no logs from fluentd will be exported?

Reply:
Ling, have you confirmed that this section does what it tries to do? As far as I'm aware, fluent logs are emitted to stdout/stderr, making this tag meaningless. If you've confirmed this behavior works as intended great, but if it doesn't add any value, I would suggest removing it.

Reply:
Yes, this indeed takes effect. I encountered infinite loops and this change fixed it.

A sample ingested fluentd log looks like below:

```
{
 insertId:  "9ppb4lg2hn1rtn"  
 logName:  "projects/stackdriver-kubernetes-1337/logs/fluent.info"  
 receiveTimestamp:  "2018-04-04T01:27:05.638928314Z"  
 resource: {
  labels: {
   cluster_name:  "stackdriver-metadata-e2e"    
   location:  "us-central1-a"    
   node_name:  "gke-stackdriver-metadata-default-pool-c622577e-vlmk"    
   project_id:  "stackdriver-kubernetes-1337"    
  }
  type:  "k8s_node"   
 }
 textPayload:  "following tail of /var/log/containers/metadata-agent-api-availability-test-p469t_metadata-agent-api-availability-test_metadata-agent-api-availability-test-ef878dbc383e0e59b558103f2acfa446c4ca8d8520695460f5764e5a5719b468.log"  
 timestamp:  "2018-04-04T01:27:04Z"  
}
```

As we can see, Fluentd logs are tagged as `fluent.info`, `fluent.error` and `fluent.warn`. 

This is because Fluentd logs are [emitted as Fluentd events directly](https://github.com/fluent/fluentd/blob/b503fdbeb670ca71cbc5de2fea43702da55b10db/lib/fluent/log.rb#L488).